File: test_cases/python/gpt/12.py
def run_tests(solution_module):
    f = solution_module.isScramble
    assert f('great', 'rgeat') is True
    assert f('abcde', 'caebd') is False
    assert f('a', 'a') is True
    assert f('abc', 'bca') is True
    assert f('abcd', 'bdac') is False

File: test_cases/c/gpt/12.c
#include <assert.h>
#include <stdbool.h>

bool isScramble(char* s1, char* s2);

int main(void) {
    assert(isScramble("great", "rgeat") == true);
    assert(isScramble("abcde", "caebd") == false);
    assert(isScramble("a", "a") == true);
    assert(isScramble("abc", "bca") == true);
    assert(isScramble("abcdefghijklmnopqrstuvwxyzabcd", "abcdefghijklmnopqrstuvwxyzabcd") == true);

    return 0;
}